fineract-provider/src/main/java/org/apache/fineract/cob/domain/LoanAccountLock.java

```diff
@@ -57,10 +60,14 @@ public class LoanAccountLock {
     @Column(name = "stacktrace")
     private String stacktrace;
 
+    @Column(name = "lock_placed_on_cob_business_date", nullable = false)
+    private LocalDate lockPlacedOnCobBusinessDate;
```

Comment:
Can't we just call it lockPlacedOnBusinessDate and use it as DateUtils.getBusinessDate()?

Reply:
Just to give a 3rd opinion: "lock_placed_on" makes more sense to me. We should not put business logic into db column naming (i know, i know...many cases its already there but...)

Reply:
we already have a "lock_placed_on", but that's the actual datetime, and we need the business date as well to be able to connect the lock and the COB execution

Reply:
i see. thanks.